packages/passkeys/passkeys/example/README.md

```diff
@@ -1,3 +1,10 @@
 # passkeys_example
 
-Demonstrates how to use the passkeys plugin.
+This example demonstrates how to use the passkeys plugin.
+We built it in a way to make it as easy to run as possible.
+
+Therefore, we have set up a pre configured instance of the relying party server using a [Corbado](https://app.corbado.com) project.
+
+The great thing about this is that you don't need to set up or configure a backend to run the example.
+The downside is that you share this relying party server with all other users of the *passkeys* plugin that also run the example.
```

Comment:
Wouldn't call it "downside", rather say: "Note, that you share this relying ...."

Also passkeys plugin vs. passkeys package here